content/docs/ops/setup/injection/index.md

```diff
@@ -11,14 +11,19 @@ time. Injection can be scoped to particular sets of namespaces using
 the webhooks `namespaceSelector` mechanism. Injection can also be
 enabled and disabled per-pod with an annotation.
 
-Whether or not a sidecar is injected is depends on three things:
+Whether or not a sidecar is injected depends on three pieces of configuration and two security rules:
 
+Configuration:
```

Comment:
These lists are displaying correctly in the generated preview: https://deploy-preview-4255--preliminary-istio.netlify.com/

I think Hugo only accepts `-` (not `*`) for list items, so it needs to be:
```
Configuration:
- webhooks `namespaceSelector`
- default `policy`
- per-pod override annotation
```

Reply:
Turns out it needs whitespace around it